cpp/include/cudf/io/detail/avro.hpp

```diff
@@ -38,7 +39,7 @@ namespace avro {
 table_with_metadata read_avro(
   std::unique_ptr<cudf::io::datasource>&& source,
   avro_reader_options const& options,
-  rmm::cuda_stream_view stream        = rmm::cuda_stream_default,
+  rmm::cuda_stream_view stream        = cudf::default_stream_value,
```

Comment:
I think this should not have a default value, it is not used. The calling code at https://github.com/rapidsai/cudf/blob/branch-22.06/cpp/src/io/functions.cpp#L158 needs to pass `cudf::default_stream_value` instead.

Reply:
Good catch!  Thanks.